src/main/java/uk/gov/justice/hmpps/prison/service/PrisonerProfileUpdateService.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.hmpps.prison.service
+
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderBookingRepository
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderRepository
+
+@Service
+class PrisonerProfileUpdateService(
+  private val offenderRepository: OffenderRepository,
+  private val offenderBookingRepository: OffenderBookingRepository,
+) {
+
+  @Transactional
+  fun updateBirthPlaceOfCurrentAlias(prisonerNumber: String, birthPlace: String?) {
+    offenderBookingRepository.findLatestOffenderBookingByNomsId(prisonerNumber)
```

Comment:
I'm not sure really why I ended up going down the explicit update route... I think I was trying to avoid the PESSIMISTIC_LOCK, but I don't think there will be a high level of contention on the record, so gone with your suggestion, added the lock, and it's much tidier now!